scripts/clear_empty_files/.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.2.2
```

Comment:
Maybe remove these scripts and squash the commits before merging because none of that will be really required in the end. Or copy them to Gist instead of deleting.